src/Libraries/Microsoft.Extensions.AI.Abstractions/ChatCompletion/IChatClient.cs

```diff
@@ -33,6 +33,20 @@ public interface IChatClient : IDisposable
     /// <param name="cancellationToken">The <see cref="CancellationToken"/> to monitor for cancellation requests. The default is <see cref="CancellationToken.None"/>.</param>
     /// <returns>The response messages generated by the client.</returns>
     /// <exception cref="ArgumentNullException"><paramref name="messages"/> is <see langword="null"/>.</exception>
+    /// <remarks>
+    /// <para>
+    /// No prompt injection defense is performed by default for data within <paramref name="messages"/>.
+    /// Applications are expected to implement guards against prompt injection attacks as needed.
+    /// </para>
+    /// <para>
+    /// There are no limits or restrictions imposed on data sizes sent to the underlying provider or returned from it
+    /// by default. Applications are expected to apply data size limits as needed.
+    /// </para>
+    /// <para>
+    /// There are no limits or restrictions imposed on the number of messages sent to the underlying provider or returned from
+    /// it by default. Applications are expected to apply rate limits on a per-user or per-session basis as needed.
+    /// </para>
+    /// </remarks>
```

Comment:
These are interfaces. We don't own the implementations (or at least not most of them). I don't know that we can make any claims about what implementations do or don't do by default with regards to limits.